packages/ckeditor5-engine/src/model/node.ts

```diff
@@ -17,9 +17,6 @@ import type Element from './element.js';
 
 import { CKEditorError, compareArrays, toMap } from '@ckeditor/ckeditor5-utils';
 
-// To check if component is loaded more than once.
-import '@ckeditor/ckeditor5-utils/src/version.js';
```

Comment:
Is it safe to remove this check for old installation methods? Do we have any other safety mechanism for version mismatch?

Reply:
According to my tests it is safe to remove.

I tested it in Vite and Webpack with two different versions of the `engine` package. In both, I removed all these side-effect imports, and the `duplicated-module` error was still thrown. It worked whether I imported the code from the main package entry points (`import { Position } from '@ckeditor/ckeditor5-engine';`) or directly from the `src' folders (`import Position from '@ckeditor/ckeditor5-engine/src/model/position.js';`).

This is because any import from `@ckeditor/ckeditor5-utils` triggers this check, and we do it all over this package.